New block images grid

kesiah opened the pull request:
Test URLs:
- Before: https://main--vg-volvotrucks-us-rd--netcentric.hlx.page/drafts/laura/
- After: https://11-our-promises--vg-volvotrucks-us-rd--netcentric.hlx.page/drafts/laura/

cogniSyb commented:
Something has gone wrong with the test URLs. But I checked out the code and could run it locally :)

Regarding alignment, the design contains heading + text that’s always over two lines. Is it possible to add a soft enter in the document so that we have at least the headings spanning two lines?

In your example I miss the 5th image, the full width landscape image. Could you add this one?

Some linting errors should be fixed as well, mostly the class selectors should be written in kebab-case.

kesiah commented:
> 
- URLS fixed!
- Added last fullwidth image, sorry I thought it was another component.
- Fixed linting issues
- Title in 2 lines: I would avoid adding `<br>` from the editor, cause in smaller viewport, we can get strange behaviour. So I can add more paddings, so the text width is decreased in bigger viewports, but currently I'm using the paddings that the figma design uses.

kesiah commented:
> Maybe it’s good to leave the content as is indeed. Even though it’s not supported in every browser, we can add `text-wrap: balance;` to both the title and the text.
> 
> I see in viewport between 460px and 744px that the images aren’t filling the cards. Could the image get a `width: 100%` in here?

done!

cogniSyb commented on blocks/images-grid/images-grid.js:
```diff
@@ -0,0 +1,52 @@
+  [...ul.children].forEach((li) => {
+    const section = document.createElement('div');
+
+    // add link to the image and move it outside of the wrapper
```
Remove comment and check the rest of the comments in the file

Lakshmishri commented on blocks/images-grid/images-grid.js:
```diff
@@ -0,0 +1,51 @@
+import { createOptimizedPicture } from '../../scripts/lib-franklin.js';
```
Can you use ```createElement ``` from script.js for creating elements. You can also pass classes to it.

Lakshmishri reviewed (changes requested):
<img width="773" alt="Screenshot 2023-08-01 at 09 05 47" src="https://github.com/Netcentric/vg-volvotrucks-us-rd/assets/4208604/abe400e2-5124-4f09-95ce-14cfb64c210d">
We have carousel for mobile, are we skipping it ?

kesiah commented:
> <img alt="Screenshot 2023-08-01 at 09 05 47" width="773" src="https://user-images.githubusercontent.com/4208604/257432928-abe400e2-5124-4f09-95ce-14cfb64c210d.png"> We have carousel for mobile, are we skipping it ?

No, the slider is there but as it's not having arrows, we are using scroll-snap to scroll in mobile. it was agreed with Syb.

Lakshmishri reviewed (changes requested):
Sorry I forgot, can we rename the files to start with v2, so we know this is a new block as we agreed.

v2-images-grid ?
